test/test-applications/regression/DuplicateTypeProxy/Properties/launchSettings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "profiles": {
+    "DuplicateTypeProxy": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "COR_ENABLE_PROFILING": "1",
+        "COR_PROFILER": "{918728DD-259F-4A6A-AC2B-B85E1B658318}",
+        "COR_PROFILER_PATH": "$(ProjectDir)$(OutputPath)profiler-lib\\Datadog.Trace.ClrProfiler.Native.dll",
+
+        "CORECLR_ENABLE_PROFILING": "1",
+        "CORECLR_PROFILER": "{918728DD-259F-4A6A-AC2B-B85E1B658318}",
+        "CORECLR_PROFILER_PATH": "$(ProjectDir)$(OutputPath)profiler-lib\\Datadog.Trace.ClrProfiler.Native.dll",
```

Comment:
These were renamed. This is something that we could automate via a tool, but not for this PR.

Is it covered in the instructions? If not could you please add it there.

Reply:
Fixed